_topics/en/lightning-gossip.md

```diff
@@ -0,0 +1,166 @@
+    url: /en/newsletters/2019/10/02/#c-lightning-3064
+
+  - title: "Blockstream Satellite begins broadcasting LN gossip data"
+    url: /en/newsletters/2021/02/17/#blockstream-satellite-broadcasting-ln-data-and-bitcoin-core-source
+
+  - title: "C-Lightning #4639 adds experimental support for gossiping liquidity advertisements"
+    url: /en/newsletters/2021/07/28/#c-lightning-4639
+
+  - title: Updated v2 gossip proposal
+    url: /en/newsletters/2022/02/23/#updated-ln-gossip-proposal
+
+  - title: Continued discussion about updated v2 gossip protocol
+    url: /en/newsletters/2022/03/30/#continued-discussion-about-updated-ln-gossip-protocol
+
+  - title: LN gossip rate limiting for use with Erlay-like set reconciliation
+    url: /en/newsletters/2022/05/04/#ln-gossip-rate-limiting
+
+  - title: LN developer meeting discussion about v2 gossip improvements
+    url: /en/newsletters/2022/06/15/#gossip-network-updates
+
+  - title: "Core Lightning #5239 improves its gossip handling code"
+    url: /en/newsletters/2022/07/13/#core-lightning-5239
+
+  - title: LN fee rate cards proposed to reduce fee-related channel update gossip
+    url: /en/newsletters/2022/09/28/#ln-fee-ratecards
+
+  - title: LDK #2198 increases the amount before gossiping that a channel is down
+    url: /en/newsletters/2023/04/26/#ldk-2198
+
+  - title: LDK #2222 allows accepting gossip without verifying it first
+    url: /en/newsletters/2023/05/03/#ldk-2222
+
+  - title: Updated channel announcements discussed during LN developer meeting
+    url: /en/newsletters/2023/07/26/#updated-channel-announcements
+
+  - title: Disclosure of two past vulnerabilities in LND gossip handling
+    url: /en/newsletters/2024/01/03/#disclosure-of-past-lnd-vulnerabilities
+
+  - title: Anonymous usage tokens proposed with potential application for LN gossip
+    url: /en/newsletters/2024/05/17/#anonymous-usage-tokens
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Unannounced channels
+    link: topic unannounced channels
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Lightning gossip** is the decentralized protocol used announce the
+  existence and settings of LN nodes and channels.
+
+---
+Many LN nodes choose to announce their existence and their channels to
+encourage other nodes to use them for routing payments.  Other nodes
+choose to remain private, using [unannounced channels][topic unannounced
+channels].
+
+As of this writing, many developers call the currently deployed
+[BOLT7][] gossip protocol "gossip v1".  One key feature of gossip v1
+is that channel announcements and updates must be signed by a key
```

Comment:
Maybe make a channel announcements topic reference

Reply:
Oh, geeze, that topic and this topic are really the same.  :man_facepalming:  I should just move this content into there.  Thanks for reminding me that existed!